CHANGES.md

```diff
@@ -62,6 +62,7 @@
 * Decreased TextSource CPU utilization by 2.3x (Java) ([#23193](https://github.com/apache/beam/issues/23193)).
 * Fixed bug when using SpannerIO with RuntimeValueProvider options (Java) ([#22146](https://github.com/apache/beam/issues/22146)).
 * Fixed issue for unicode rendering on WriteToBigQuery ([#10785](https://github.com/apache/beam/issues/10785))
+* Remove obsolete variant of BigQuery Write ([#23564](https://github.com/apache/beam/issues/23564)).
```

Comment:
Maybe comment that old code using this variant will redirect to the new variant with a warning.  This isn't a breaking change right?

Reply:
The same line added in #23557 introduces a conflict. Could just change this line to
```
Remove obsolete variant of BigQuery Read and Write ([#23559](https://github.com/apache/beam/issues/23559)).
```
during rebase.

Reply:
Yeah, I merged these lines in the changelog.